Deactivate automatic source maps upload for codepush (#4441)
getsentry/sentry-docs: src/platforms/react-native/manual-setup/codepush.mdx

To use Sentry together with CodePush, pass `release` and `dist` to `Sentry.init`. Our suggestion is to store them in an `app.json`, or you can just use the `package.json`. These values need to be unique to each version of your codebase and match the version on the source maps exactly, or they might not be symbolicated.

> When you set a custom release or manually upload source maps, you will need to disable the automatic source map upload script, check out the [Disable the automatic source maps upload script](/platforms/react-native/sourcemaps/#2-disable-the-automatic-source-maps-upload-script) docs.

Author: ```suggestion
When you set a custom `release` and `dist` or manually upload source maps, you will need to disable the automatic source map upload script, check out the [Disable the automatic source maps upload script](/platforms/react-native/sourcemaps/#2-disable-the-automatic-source-maps-upload-script) docs.
```

> When you set a custom `release` and `dist` or manually upload source maps, you will need to disable the automatic source map upload script, check out the [Disable the automatic source maps upload script](/platforms/react-native/sourcemaps/#2-disable-the-automatic-source-maps-upload-script) docs.

Reviewer: ```suggestion
When you set a custom `release` and `dist`, or manually upload source maps, you will need to disable the automatic source map upload script. Learn more in the [Disable the automatic source maps upload script](/platforms/react-native/sourcemaps/#2-disable-the-automatic-source-maps-upload-script) documentation.
```

> For more information on the `release` and `dist` parameters, see the section on [Source Maps](/platforms/react-native/sourcemaps/).

Reviewer: ```suggestion
For more information on the `release` and `dist` parameters, check out the section on [Source Maps](/platforms/react-native/sourcemaps/).
```
